.github/workflows/deployment.yml

```diff
@@ -45,7 +45,17 @@ jobs:
         with:
           version: 'latest'
 
+      - name: Cache dependencies
```

Comment:
`actions/setup-node@v3` already caches the dependencies, so why do we need to do this? :thinking:

Reply:
Symlinks to the packages built are not included in the node cache, and these are needed for the typedoc build. The way this worked prior to this change, was to include the symlinks in the artifact meant for publishing, which poluted the `build-release` artifact.